src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -92,17 +93,21 @@ class PeerManagerActor(
       val peerAddresses = outgoingPeersAddresses(peers)
 
       val nodesToConnect = nodesInfo
-        .filterNot{ discoveryNodeInfo =>
+        .filterNot { discoveryNodeInfo =>
           val socketAddress = discoveryNodeInfo.node.tcpSocketAddress
           peerAddresses.contains(socketAddress) || isBlacklisted(PeerAddress(socketAddress.getHostString))
         } // not already connected to or blacklisted
         .take(peerConfiguration.maxOutgoingPeers - peerAddresses.size)
 
+      NetworkMetrics.DiscoveredPeersSize.set(nodesInfo.size)
```

Comment:
Aren't `nodes` the discovered ones? They have been some of them were removed when getting `nodesInfo`

Reply:
Oh you're right, I misread